api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -94,18 +95,22 @@
      * The configuration value is not guaranteed to be cached by the implementation, and may be expensive
      * to compute; therefore, if the returned value is intended to be frequently used, callers should consider storing
      * rather than recomputing it.
+     * <p>
+     * The result of this method is identical to the result of calling {@code getOptionalValue(propertyName, propertyType).get()}.
+     * In particular, if the value element of the given property name does not exist, a {@link NoSuchElementException} is thrown.
```

Comment:
How about:
If the given property name or the value element of the give property name does not exist, a {@link NoSuchElementException} is thrown.